docs/README.md

```diff
@@ -1,8 +1,8 @@
 ![Fuel Logo](./img/fuel.png)
 
-The Fuel Indexer acts as a standalone binary that can be used to index various outputs of a [Fuel node](https://github.com/FuelLabs/fuel-core). These indexable outputs include blocks, transactions, [transaction receipts](https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/tx_format.md), and a Fuel node's general state, allowing for high-performance read-only access to the blockchain for advanced dApp use-cases.
+The Fuel indexer acts as a standalone binary that can be used to index various outputs of a [Fuel node](https://github.com/FuelLabs/fuel-core). These indexable outputs include blocks, transactions, [transaction receipts](https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/tx_format.md), and a Fuel node's general state, allowing for high-performance read-only access to the blockchain for advanced dApp use-cases.
```

Comment:
Non-blocking: do we want to change "binary" to "service" to match the change in `README.md`?